app/src/main/java/com/github/featuredetectandroid/ui/theme/Type.kt

```diff
@@ -10,6 +10,6 @@ val Typography = Typography(
     body1 = TextStyle(
         fontFamily = FontFamily.Default,
         fontWeight = FontWeight.Normal,
-        fontSize = 16.sp
+        fontSize = 20.sp
```

Comment:
I think you need to set it back to the default value because `body1` is now as big as `h6`

Reply:
fixed